docs/userGuide/contentAuthoring.md

```diff
@@ -7,11 +7,7 @@
 
 <div class="website-content">
 
-# Content authoring
-
-### General writing guide
-
-#### Mixing HTML with Markdown
+# Content Authoring

```

Comment:
Just a side note: Some pages seems to have caps for all words but some pages don't (see `Including Contents` and `Known Problems`)

I think in a future PR could do some more minor cleanups in the documentation.